build.gradle

```diff
@@ -82,12 +86,3 @@ dockerPush.onlyIf {
     versionDetails().isCleanTag
 }
 
-publishing {
```

Comment:
We're no longer publishing an artifact named `com.palantir.cassandra:palantir-cassandra` - what should we be using instead?